packages/design-system-docs/src/pages/startup/child-design-systems.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Child Design Systems
+weight: 6
+---
+
+The CMS design system (CMSDS) provides a set of components and styles to meet many design and development needs. However, there are cases where product teams need to modify a design system beyond [theming]({{root}}/startup/theming/) and publish the new system for other teams to consume.
```

Comment:
Consider updating to

The CMS design system (CMSDS) provides a set of components and styles to meet many design and development needs. However, there are cases where product teams need to modify the CMSDS beyond [theming]({{root}}/startup/theming/) and publish a child design system for other product teams to consume.